### ISSUE TITLE
mpii submission

### ISSUE THREAD

**Issue by yellowcheng** [leoxiaobin/deep-high-resolution-net.pytorch#12]
hi, i use test.json to creat a [7247. 16. 2] pred.mat, can i submitt pred.mat to mpii official website to have a test result?

**Comment by leoxiaobin** [comment]
You should convert to MPII's format following his instruction at http://human-pose.mpi-inf.mpg.de/#evaluation  for submission.

**Comment by laowangFish** [comment]
> You should convert to MPII's format following his instruction at http://human-pose.mpi-inf.mpg.de/#evaluation for submission.

Thank you for your attention. I don't understand the scale param in the valid.json/test.json of MPII dataset. The explaination of the office document is ".scale - person scale w.r.t. 200 px height" . Are they  the same?

**Comment by aliYaHf** [comment]
@yellowcheng 
Hi！ Are you using the official data of the MPII dataset, or the processed data provided by Xiao, how is the training result?

**Comment by yellowcheng** [comment]
@aliYaHf 
yes, and the result on mpii val is the same as the paper, but i still can't figure out how to prepare my test result and submit to the mpii website

**Comment by aliYaHf** [comment]
@yellowcheng Are you using the parameters given by the author in the paper? I wonder why my training is so low.

**Comment by 738654805** [comment]
@yellowcheng Have you solve this problem? I meet the same problem.

**Comment by U-C-J** [comment]
Same problem. 
In test.json provided by HRnet, the test.json size is 7247. 
But in https://github.com/anibali/eval-mpii-pose, the test.h5 size is 11731.
Also, the center and scale are different in test.json and test.h5 in that post. 
Now I am really confused. Did anyone figure it out? Thank you.

**Comment by Github3G** [comment]
> Same problem.
> In test.json provided by HRnet, the test.json size is 7247.
> But in https://github.com/anibali/eval-mpii-pose, the test.h5 size is 11731.
> Also, the center and scale are different in test.json and test.h5 in that post.
> Now I am really confused. Did anyone figure it out? Thank you.

Do you have resolved this problem?  I meet same proble, and the center and scale are differnet, but i use the h5 to test for submitt. at the other questiion [https://github.com/leoxiaobin/deep-high-resolution-net.pytorch/issues/51](url), one person think "center" values are annotated manually

**Comment by PCRKTY** [comment]
> hi, i use test.json to creat a [7247. 16. 2] pred.mat, can i submitt pred.mat to mpii official website to have a test result?

May I ask you how to use the test.json to create a [7247 samples] pred.mat?

**Comment by MaxChu719** [comment]
> > Same problem.
> > In test.json provided by HRnet, the test.json size is 7247.
> > But in https://github.com/anibali/eval-mpii-pose, the test.h5 size is 11731.
> > Also, the center and scale are different in test.json and test.h5 in that post.
> > Now I am really confused. Did anyone figure it out? Thank you.
> 
> Do you have resolved this problem? I meet same proble, and the center and scale are differnet, but i use the h5 to test for submitt. at the other questiion [https://github.com/leoxiaobin/deep-high-resolution-net.pytorch/issues/51](url), one person think "center" values are annotated manually

I uses the test.h5 to generate the 11731 test results and the PCKh score i got is only 89.9 instead of 92.3 reported in the paper.
I have implemented the multi-scale test on the valid set and I got pckh  90.75 so the multi-scale output should be fine.

**Comment by STRUGGLE1999** [comment]
> hi, i use test.json to creat a [7247. 16. 2] pred.mat, can i submitt pred.mat to mpii official website to have a test result?



> hi, i use test.json to creat a [7247. 16. 2] pred.mat, can i submitt pred.mat to mpii official website to have a test result?

Hello, Could you tell me how to convert pred.mat to json file for visualision ?

**Comment by STRUGGLE1999** [comment]
> You should convert to MPII's format following his instruction at http://human-pose.mpi-inf.mpg.de/#evaluation for submission.

Hello, Could you tell me how to get a test result for mpii, I get a pred.mat, but couldn't visualize it.
